src/transformers/models/beit/modeling_beit.py

```diff
@@ -455,6 +486,16 @@ def __init__(self, config: BeitConfig, window_size: tuple) -> None:
         )  # 2*Wh-1 * 2*Ww-1, nH
         # cls to token & token 2 cls & cls to cls
 
+        # # for backwards compatibility purposes, we still set the buffer even though we're not using it
+        relative_position_index = self.generate_relative_position_index_original(window_size)
+        self.register_buffer("relative_position_index", relative_position_index, persistent=False)
+        self.relative_position_indices = {}
+
+    def generate_relative_position_index_original(self, window_size: Tuple[int, int]):
```

Comment:
Having these two methods is a good indication that we shouldn't be doing what we're doing. The modeling file shouldn't become a place to keep multiple possible code paths. If adding zoedepth means the beit model is affected substantially, then we should think about other ways to integrate the model